src/app/onboarding/components/Map/stories/Map.stories.tsx

```
@@ -0,0 +1,58 @@
+// Map.stories.tsx
+import type { Meta, StoryObj } from "@storybook/nextjs";
+import { useState } from "react";
+import { Map } from "../Map";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)